.ci/containers/terraform-tester/test_terraform.sh

```diff
@@ -29,7 +29,7 @@ pushd $local_path
 echo "Checking for modified go files"
 # get the names of changed files and look for go files
 # (ignoring "no matches found" errors from grep)
-gofiles=$(git diff --name-only HEAD~1 | { grep "\.go$" || test $? = 1; })
+gofiles=$(git diff --name-only HEAD~1 | { grep -e "\.go$" -e "go.mod$" -e "go.sum$" || test $? = 1; })
```

Comment:
This change didn't appear to take! You'll probably have to build the image from your branch: https://github.com/GoogleCloudPlatform/magic-modules/tree/master/.ci/containers#updating-a-docker-image

Reply:
Sure. Will do.